providers/ms365/resources/ms365.lr

```diff
@@ -119,16 +119,29 @@ microsoft.users {
   search string
 }
 
-// A container resource for identity and access policies that can be filtered
+// A container resource for identity and access management
 microsoft.identityAndAccess {
-  []microsoft.identityAndAccess.policy
+  // Get the instances of role eligibilities
```

Comment:
this pr needs a rebase - `  roleEligibilityScheduleInstances() []microsoft.identityAndAccess.roleEligibilityScheduleInstance` already exists on main